hudi-utilities/src/test/java/org/apache/hudi/utilities/TestHoodieDeltaStreamer.java

```diff
@@ -413,37 +412,33 @@ public void testKafkaConnectCheckpointProvider() throws IOException {
     HoodieTestDataGenerator dataGenerator = new HoodieTestDataGenerator();
     Helpers.saveParquetToDFS(Helpers.toGenericRecords(dataGenerator.generateInserts("000", 100)), new Path(filePath));
     HoodieDeltaStreamer deltaStreamer = new HoodieDeltaStreamer(cfg, jsc, dfs, hdfsTestService.getHadoopConf(), props);
-    assertEquals(deltaStreamer.getConfig().checkpoint, "kafka_topic1,0:200");
+    assertEquals("kafka_topic1,0:200", deltaStreamer.getConfig().checkpoint);
   }
 
   @Test
   public void testPropsWithInvalidKeyGenerator() throws Exception {
-    try {
+    Exception e = assertThrows(IOException.class, () -> {
       String tableBasePath = dfsBasePath + "/test_table";
       HoodieDeltaStreamer deltaStreamer =
           new HoodieDeltaStreamer(TestHelpers.makeConfig(tableBasePath, Operation.BULK_INSERT,
               Collections.singletonList(TripsWithDistanceTransformer.class.getName()), PROPS_FILENAME_TEST_INVALID, false), jsc);
       deltaStreamer.sync();
-      fail("Should error out when setting the key generator class property to an invalid value");
-    } catch (IOException e) {
-      // expected
-      LOG.error("Expected error during getting the key generator", e);
-      assertTrue(e.getMessage().contains("Could not load key generator class"));
-    }
+    }, "Should error out when setting the key generator class property to an invalid value");
+    // expected
+    LOG.debug("Expected error during getting the key generator", e);
+    assertTrue(e.getMessage().contains("Could not load key generator class"));
```

Comment:
Changing log level from error to debug. It is a good practice to not print anything unless test case failed thus we tend to mute the output here when it's passing. Same to other similar changes below.